SECURITY.md

```diff
@@ -4,9 +4,9 @@
 
 | Version | Supported          |
 | ------- | ------------------ |
-| 21      | :white_check_mark: |
-| 20.1    | :white_check_mark: |
-| < 20.1  | :x:                |
+| 22      | :white_check_mark: |
+| 21.1    | :white_check_mark: |
```

Comment:
I think 21.1 should not be supported because we have fork `withdrawal` which is not implemented for 21.1

Reply:
Resolved in latest push